samples/GSC/shared.gsh

```diff
@@ -0,0 +1,935 @@
+#define PLAYER_1 0

```

Comment:
Are you able to find a smaller sample? This file is mostly filled with `define` lines which don't differ by very much syntactically.